packages/flame_studio/lib/src/widgets/scaffold.dart

```diff
@@ -6,12 +6,13 @@ import 'package:flutter/widgets.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
 class Scaffold extends ConsumerWidget {
```

Comment:
We shouldn't re-use names for such basic Flutter widgets, that will be super confusing.

Reply:
Must have missed this on the last PR.

Reply:
Renamed into `UiScaffold`.